rakelib/plugin.rake

```diff
@@ -61,6 +61,8 @@ namespace "plugin" do
     Dir["logstash-core*.gem"].each do |gem|
       rm(gem)
     end
```

Comment:
Well, it will check all the possible path, it will be probably faster if we use a more specific path?